### DISCUSSION TITLE
can't run bladebit

### DISCUSSION THREAD

**Discussion by olivierLA75** [Chia-Network/bladebit#64] [Q&A]
Hi, 
I installed make and then followed all the steps. I have the bladebit folder and I have built the bras library but I can't find how to start bladebit. when I input bladebit -h in terminal it says command not found and when I input ./bladebit -h it says ./bladebit Is a directory. please help. thanks!

**Comment by harold-b** [comment]
The binary can be found under the .bin/<cofig_name> directory. I've just updated the readme with more explicit instructions, please have a look.

**Comment by olivierLA75** [comment]
Thank you. I believe I have installed everything correctly but can't find the .bin, any idea of what can be the issue here?

**Reply by olegin77** [reply]
I have the same
Problem , maybe version of gcc , anybody can give version of gif wich supports bls ?

**Reply by harold-b** [reply]
> Thank you. I believe I have installed everything correctly but can't find the .bin, any idea of what can be the issue here?

Folders that start with `.` are hidden in *nix platforms. So you won't see it unless you use something like `ls -la`. But they are there as long as your compilation did not yield any errors.

> Problem , maybe version of gcc , anybody can give version of gif wich supports bls ?

I used GCC 7.x on CentOS and 9.3.0 on Ubuntu.

**Comment by olivierLA75** [comment]
I reinstalled everything and it works now. Does anyone as an exemple of the command to launch the plotting process?

**Reply by olegin77** [reply]
Could you give version of all libraries which you installed and especially GCC version . Thanks a lot

**Reply by olivierLA75** [reply]
I was wrong it doesn't work. can't find the .bin.

**Comment by olivierLA75** [comment]
Is the file supposed to be bladebit.bin ?

**Reply by harold-b** [reply]
No, it is just `bladebit`.

 `.bin` is just the name of the folder inside which you can find it. See https://github.com/harold-b/bladebit#usage

**Comment by olivierLA75** [comment]
Got. I don’t have it. I guess I am going to try one more time

**Comment by olivierLA75** [comment]
Has anyone succeeded in having Bladebit work on Ubuntu 20.04? I tried everything and still no .bin file :(

**Reply by harold-b** [reply]
Are you getting compilation errors?

**Reply by olivierLA75** [reply]
actually I think its working now. could you share an example of the command to launch plotting, not sure in which order to write it. thanks!
